contracts/l1/Bridge.sol

```diff
@@ -40,6 +40,16 @@ contract Bridge is IBridge, VersionedInitializable {
         _;
     }
 
+    //returns the available rewards on the bridge : claimed+pending before consuming any bridge_rewards message from l2, otherwise the user loses all his rewards after the message is consumed and the _transfer reverts
+    function getAvailableRewards() external view returns (uint256) {
```

Comment:
this function should be internal (or public) since we call it from another function of the contract